<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordEditForm</class>
 <widget class="QWidget" name="RecordEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>370</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Product</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_product">
        <item>
         <property name="text">
          <string>Milk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bread</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Meat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tomatoes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Potation</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Company_ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spb_company_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txt_details"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>318</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bnt_apply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
